ide/html.parser/external/htmlparser-01-fix-maven-build.patch

```diff
@@ -45,7 +45,7 @@ diff --git a/src/nu/validator/htmlparser/dom/HtmlDocumentBuilder.java b/src/nu/v
 --- a/src/nu/validator/htmlparser/dom/HtmlDocumentBuilder.java
 +++ b/src/nu/validator/htmlparser/dom/HtmlDocumentBuilder.java
 @@ -377,26 +377,26 @@ public class HtmlDocumentBuilder extends
-     public void setTransitionHander(TransitionHandler handler) {
+     public void setTransitionHandler(TransitionHandler handler) {
```

Comment:
You're changing a diff file here, are the offsets still correct? Or will this patch file be broken after your change?

Reply:
This file
"Fix error in maven build when run on JDK8"
and therefore should be removed

Reply:
file reverted in 537ad69d to remove in feature

Reply:
> This file "Fix error in maven build when run on JDK8" and therefore should be removed

Don't guess. Especially not when you are fixing "typos".

Reply:
ok, reverted